### PR TITLE
feat(cketh): threshold consensus strategy for HTTPs outcalls

### PR DESCRIPTION
**Pull request by gregorydemay**
Use a 3-out-of 4 consensus strategy when querying the EVM RPC canister to avoid a provider being able to block the ckETH minter.

### CODE REVIEW

File: rs/ethereum/cketh/minter/src/eth_rpc_client/mod.rs [hunk 1]
```diff
@@ -56,25 +56,40 @@ impl EthRpcClient {
     pub fn from_state(state: &State) -> Self {
         let mut client = Self::new(state.ethereum_network());
         if let Some(evm_rpc_id) = state.evm_rpc_id {
-            const MIN_ATTACHED_CYCLES: u128 = 400_000_000_000;
+            const MIN_ATTACHED_CYCLES: u128 = 500_000_000_000;
 
             let providers = match client.chain {
                 EthereumNetwork::Mainnet => EthereumProvider::evm_rpc_node_providers(),
                 EthereumNetwork::Sepolia => SepoliaProvider::evm_rpc_node_providers(),
             };
```
Comment (gregorydemay) [inline rs/ethereum/cketh/minter/src/eth_rpc_client/mod.rs:64]: @THLO @ninegua: we currently manually specify the providers, but I think we should not and let the EVM RPC canister decide which 4 providers to use. For this I first need this other [PR](https://github.com/internet-computer-protocol/evm-rpc-canister/pull/305), so I would do it in another PR.

**Review by THLO — APPROVED** [review]

**Review by ninegua — APPROVED** [review]
